inginious/frontend/common/static/css/bootstrap-modal-fullscreen.css

```diff
@@ -0,0 +1,46 @@
+/* Copy-paste from http://www.minimit.com/demos/bootstrap-3-transparent-and-fullscreen-modals */
```

Comment:
Mea culpa; it was the "HTTPS anywhere" plugin that forced the usage of https.

Where do you see it's MIT?